Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -939,6 +953,35 @@ class BrowserViewController: UIViewController {
         let ua = newRequest.valueForHTTPHeaderField("User-Agent")
         webView.customUserAgent = ua != UserAgent.defaultUserAgent() ? ua : nil
     }
+
+    private func updateFindInPageVisibility(visible visible: Bool) {
+        if visible {
+            if findInPageBar == nil {
+                let findInPageBar = FindInPageBar()
+                self.findInPageBar = findInPageBar
+                findInPageBar.delegate = self
+                findInPageContainer.addSubview(findInPageBar)
+
+                findInPageBar.snp_makeConstraints { make in
+                    make.edges.equalTo(findInPageContainer)
+                    make.height.equalTo(UIConstants.ToolbarHeight)
+                }
+
+                updateViewConstraints()
+
+                findInPageBar.layoutIfNeeded()
```

Comment:
Usually better to call `setNeedsLayout` instead of `layoutIfNeeded()` so the layout will occur on the next layout pass instead of forcing one.